### ISSUE TITLE
TypeError: webpack.DefinePlugin is not a constructor

### ISSUE THREAD

**Issue by jasonlfunk** [JeffreyWay/laravel-mix#2750]
-   Laravel Mix Version: 6.0.8 (`npm list --depth=0`)
-   Node Version (`node -v`): v12.18.3
-   NPM Version (`npm -v`): 6.14.6
-   OS: MacOS 11.1

### Description:

Version 6.0.8 broke `mix.webpackConfig`.

```
[webpack-cli] TypeError: webpack.DefinePlugin is not a constructor
    at /Users/jasonlfunk/Workspace/Source/Work/Legnd/fablink/webpack.mix.js:18:13
    at /Users/jasonlfunk/Workspace/Source/Work/Legnd/fablink/node_modules/laravel-mix/src/components/WebpackConfig.js:6:52
    at /Users/jasonlfunk/Workspace/Source/Work/Legnd/fablink/node_modules/laravel-mix/src/Dispatcher.js:34:69
    at Array.map (<anonymous>)
```

```
mix.webpackConfig(webpack => {
    return {
        plugins: [
            new webpack.DefinePlugin({
                "process.env.ASSET_PATH": JSON.stringify(ASSET_URL)
            })
        ],
```

**Comment by jasonlfunk** [comment]
v6.0.6 works fine.

v6.0.7 gives a different error which seems to have been fixed by d2554ed842c79b009c0846c7a77c1188cc5b2995

**Comment by AhsanAbrar** [comment]
yes getting the same error

```js
  .webpackConfig(webpack => {
    return {
      resolve: {
        extensions: ['.ts', '.tsx', '.js', '.jsx', '.vue']
      },
      plugins: [
        new webpack.DefinePlugin({
          __VUE_OPTIONS_API__: true,
          __VUE_PROD_DEVTOOLS__: false
        })
      ]
    };
  })
```
